applications/portfolioDrafts/deletePortfolioDraft.ts

```diff
@@ -0,0 +1,46 @@
+import { DeleteCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { parse } from "uuid";
+import { dynamodbClient as client } from "./utils/dynamodb";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+// const PRIMARY_KEY = process.env.PRIMARY_KEY || "";
+
+/**
+ * Deletes a Portfolio Draft
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (portfolioDraftId == null) {
+    return { statusCode: 400, body: "Bad Request. A Portfolio Draft ID must be specified in URL path." };
+  }
+
+  if (!parse(portfolioDraftId)) {
+    return { statusCode: 400, body: "Bad Request. Portfolio Draft ID must be a UUID compliant with RFC 4122." };
+  }
```

Comment:
I am not sure that I like exposing the implementation detail that an ID must be a valid UUID. I think we can still perform this check but maybe return a `404` and a say that a Portfolio Draft with that ID doesn't exist (because we know it won't)?